gensim/models/fasttext.py

```diff
@@ -759,7 +835,8 @@ def load_fasttext_format(cls, model_file, encoding='utf8'):
 
         Notes
         ------
-        Due to limitations in the FastText API, you cannot continue training with a model loaded this way.
+        This function effectively ignores `.vec` output file.
```

Comment:
What is the `.vec` for, if it's ignored?

Reply:
OK, I added an explanation.